tests/testsuite/cargo_new/inherit_workspace_package_table.in/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[workspace]
+members = [
+  "foo",
```

Comment:
All tests currently specify `foo` before it was created. While this may happen sometimes, there should be tests that use glob syntax as well as a test for what happens when it is not a member of the workspace.

Reply:
We should probably switch the tests to globs.

Without `foo` already existing, I believe this would be a broken `Cargo.toml`.  This makes the test brittle for if we make other changes to `cargo new` (unless we already promise to support broken `Cargo.toml` files)

Reply:
> Without `foo` already existing, I believe this would be a broken `Cargo.toml`. This makes the test brittle for if we make other changes to `cargo new` (unless we already promise to support broken `Cargo.toml` files)

It is not a broken `Cargo.toml`, it is a normal `Cargo.toml`. But it will cause a warning that reminds the user put the `foo` package into the members or exclude it.

Reply:
Tests updated and add a new test for the new package is not a workspace member.